tests/drum/test_other_cases.py

```diff
@@ -463,3 +468,36 @@ def test_drum_exits_code_when_custom_task_schema_validation_exception_is_raised(
                 with pytest.raises(SystemExit) as ex:
                     main()
                 assert ex.value.code == ExitCodes.SCHEMA_VALIDATION_ERROR.value
+
+    def test_model_metadata_validation_fails__with_hyper_parameters_errors(self, tmp_path):
```

Comment:
I'd add some more tests to cover more use-cases at least for the multi hyper parameter type.

Reply:
This testing was designed to verify `DrumCommonException` is properly raised when hyper parameter schema validation fails. For testing different use cases of hyper param validation, they will be in test_units. A testing was added to improve the testing coverage of multi-type hyper parameter:
https://github.com/datarobot/datarobot-user-models/blob/e7ba94491c54a5091532ae3a8529eb46f06b4cb5/tests/drum/test_units.py#L769-L775